cli/src/main/scala/scalaxb/compiler/wsdl11/Driver.scala

```diff
@@ -161,7 +163,8 @@ class Driver extends Module { driver =>
          case ImportDecl(_, Some(schemaLocation: String)) => schemaLocation
         }})
     val includeLocations: Seq[String] = schemaLite flatMap { schemaLite =>
-      schemaLite.includes map { _.schemaLocation }
+      schemaLite.includes map { include =>
+        Paths.get(alocation.getPath.substring(0, alocation.getPath.lastIndexOf('/') + 1) + include.schemaLocation).normalize().toString() }
```

Comment:
Is this logic repeated twice? If so it should be refactored into Module object or wherever it makes sense.